tests/system/migrations/test_0042_update_workflow_timestamp.py

```diff
@@ -0,0 +1,126 @@
+def test_migration(write, finalize, assert_model):
+    write(
+        {
+            "type": "create",
+            "fqid": "motion/1",
+            "fields": {
+                "a": 1,
+                "created": 1686740704,
+            },
+        },
+        {
+            "type": "create",
+            "fqid": "motion_workflow/1",
+            "fields": {"b": 1, "set_created_timestamp": True},
```

Comment:
there is no set_created_timestamp in motion_workflow

Reply:
Ah, it is too hot, stupid mistake.

Reply:
I will fix it.